dts/bindings/radio_fem/skyworks,sky66112-11.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+
+description: >
+    This is a representation of the Skyworks SKY66112-11 Radio Front-End module.
+
+compatible: "skyworks,sky66112-11"
+
+include: base.yaml
+
+properties:
+    ctx-gpios:
+        type: phandle-array
+        required: false
+        description: >
+            Gpio of the SOC controlling CTX pin of the SKY66112-11
+
+    crx-gpios:
+        type: phandle-array
+        required: false
+        description: >
+            Gpio of the SOC controlling CRX pin of the SKY66112-11
+
+    ctx-settle-time-us:
+        type: int
+        default: 23
+        description: >
+            Settling time in microseconds from activation of CTX to trasnsmit.
+
+    crx-settle-time-us:
+        type: int
+        default: 5
+        description: >
+            Settling time in microseconds from activation of CRX to receive.
+
+    tx-gain-db:
+        type: int
+        default: 22
+        description: >
+            TX gain of the PA amplifier of SKY66112-11 in dB.
+
+    rx-gain-db:
+        type: int
+        default: 11
+        description: >
+            RX gain of the LNA amplifier of SKY66112-11 in dB.
```

Comment:
The zephyr policy is that all defaults must be documented with a justifiable rationale.

For example, if these are the chip's power-on-reset values, and they can be individually configured away from these defaults without affecting other options, the descriptions should say that those are the power-on-reset values.

But if the power on reset values must be configured in a group, then the policy is to make the properties all required.

For example, if I can't change `rx-gain-db` without also having to change `tx-gain-db`, then there should be no default values, and the properties should just be required, with an explanation of some recommended defaults in the top-level description for the binding (right after "SKY66112-11 Radio Front-End module.").

Reply:
I added a description of defaults